atomix/cluster/src/main/java/io/atomix/cluster/protocol/SwimMembershipProtocol.java

```diff
@@ -624,7 +645,7 @@ private CompletableFuture<Boolean> requestProbe(
         .exceptionally(e -> false)
         .thenApply(
             succeeded -> {
-              LOGGER.debug(
+              PROBE_LOGGER.info(
```

Comment:
:question: I don't understand why this is info. There is no real state change here or useful information for a user here. It's useful for debugging only, no?

Reply:
Agreed. Debug is enough here.